.travis.yml

```diff
@@ -12,6 +12,18 @@ matrix:
         - python: "3.4"
         - python: "pypy3"
         - python: "pypy"
+before_install:
+    - identify -version
+    - convert -list format
+    - sudo apt-get remove imagemagick
+    - wget http://www.imagemagick.org/download/releases/ImageMagick-6.9.1-10.tar.xz
+    - tar xf ImageMagick-6.9.1-10.tar.xz
+    - cd ImageMagick*
+    - ./configure --disable-shared --with-openjp2=yes --with-webp=yes --prefix=/usr
+    - make
+    - sudo make install
```

Comment:
@guilhermef That might be the better way to do this. Though I'm sure it has to get the jpeg2000 support from somewhere. I haven't worked with it before so I guess I'll have to look around. Will read the docs.